src/lib.rs

```diff
@@ -1912,6 +1912,19 @@ impl Build {
                         cmd.push_cc_arg("-fno-plt".into());
                     }
                 }
+                if target.contains("wasm32-wasi") {
```

Comment:
Maybe just:

```suggestion
                if target.contains("-wasi") {
```

To support `wasm64-wasi` if that ever arrives?